roles/mailserver/files/docker-compose.yml

```diff
@@ -138,9 +153,19 @@ services:
       - mailserver
       - postfixadmin
       - rainloop
+{% if www_is_opencart %}
+      - /mnt/docker/opencart:/opencart
+{% endif %}
+    depends_on:
+      - mailserver
+      - postfixadmin
+      - rainloop
 {% if www_is_wordpress %}
       - wordpress
 {% else %}
+{% if www_is_opencart %}
+      - opencart
+{% else %}
       - contact
 {% endif %}
```

Comment:
The if/else logic here does not seem right. 

Looks like it should be something along the lines of:
```yaml
{% if www_is_wordpress %}
    - wordpress
{% elif www_is_opencart %}
    - opencart
{% else %}
    - contact
{% endif %}
```
see http://jinja.pocoo.org/docs/2.10/templates/#builtin-filters